packages/cudos-utilities/contract-services.js

```diff
@@ -0,0 +1,123 @@
+const {
```

Comment:
I would rename the file to `contract-utils`

Reply:
isn't it the full name `contract-utilities` better? (as in the folder name)

Reply:
for me, no, `utils` is way more common name in my opinion